widget/entry.go

```diff
@@ -116,6 +117,7 @@ type Entry struct {
 	CursorRow, CursorColumn int
 
 	focused bool
+	fyne.Window
```

Comment:
I think this is from the previous version right?

Reply:
Fixed in 3929f54

Reply:
Right :)